app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -39,22 +48,14 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.reflect.ClassPath;
 import com.google.common.reflect.ClassPath.ClassInfo;
-import org.joda.time.LocalDate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import org.sagebionetworks.bridge.config.BridgeConfig;
-import org.sagebionetworks.bridge.config.BridgeConfigFactory;
-import org.sagebionetworks.bridge.config.Environment;
-import org.sagebionetworks.bridge.exceptions.BridgeInitializationException;
 
 public class DynamoInitializer {
 
     private static Logger logger = LoggerFactory.getLogger(DynamoInitializer.class);
 
-    // DynamoDB Free tier
-    private static final Long READ_CAPACITY = Long.valueOf(25);
-    private static final Long WRITE_CAPACITY = Long.valueOf(25);
+    // Default capacities are the DynamoDB free tier
+    static final long READ_CAPACITY = 25;
+    static final long WRITE_CAPACITY = 25;
```

Comment:
Can we rename these to DEFAULT_READ_CAPACITY and DEFAULT_WRITE_CAPACITY so it's clear that's what they are?

Also, can we make them public for code cleanliness?